Assets/Scripts/StandardSamples/UI/Common/SampleMenu.cs

```diff
@@ -285,9 +302,31 @@ public virtual void Hide()
             // Flag as hidden.
             Hidden = true;
 
+            HideInternal();
+
             Log($"Hide() completed");
         }
 
+        /// <summary>
+        /// If this menu is shown through <see cref="Show"/> successfully, then
+        /// this method is run so that implementing child classes can respond
+        /// to becoming available.
+        /// </summary>
+        protected virtual void ShowInternal()
+        {
+
```

Comment:
I think you should add a comment to the empty function that indicates that the default behavior is to take no action.